services/httpd/handler.go

```diff
@@ -319,8 +320,8 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user *meta.
 
 	// Execute query.
 	w.Header().Add("Connection", "close")
-	w.Header().Add("content-type", "application/json")
-	readonly := r.Method == "GET" || r.Method == "HEAD"
+	w.Header().Add("Content-Type", "application/json")
+	readonly := r.Method == "GET"
```

Comment:
Just seems redundant to me, and gives the impression it's possible. I don't mind putting it back though.